src/Symfony/Component/ErrorHandler/Tests/phpt/decorate_exception_handler_lt_74.phpt

```diff
@@ -0,0 +1,56 @@
+--TEST--
+Test catching fatal errors when handlers are nested (PHP < 7.4)
+--SKIPIF--
+<?php if (PHP_VERSION_ID >= 70400) die('skip: Only for PHP < 7.4.'); ?>
+--INI--
+display_errors=0
+--FILE--
```

Comment:
Can't extract this in a separate file without altering the composer.json because of the return type patching on 7.4.